app/models/ability.rb

```diff
@@ -137,6 +137,8 @@ def signed_in_with_organizer_role(user)
     can :manage, Sponsor, conference_id: conf_ids_for_organizer
     can :manage, SponsorshipLevel, conference_id: conf_ids_for_organizer
     can :manage, Ticket, conference_id: conf_ids_for_organizer
+    can [:index, :create], Comment, commentable_type: 'Event',
```

Comment:
You are authorizing create action but to which controller action does it correspond to?